effekt/jvm/src/test/scala/effekt/core/PolymorphismBoxingTests.scala

```diff
@@ -18,6 +18,8 @@ abstract class AbstractPolymorphismBoxingTests extends CorePhaseTests(Polymorphi
     "boxInt" -> Id("boxInt"),
     "unboxInt" -> Id("unboxInt"),
     "unboxString" -> Id("unboxInt"),
+    "coerceIntPos" -> Id("@coerceIntPos"),
+    "coercePosInt" -> Id("@coercePosInt"),
```

Comment:
The problem is here. `TestRenamer` *only* renames non-toplevel *bound* names. These names should be the ones generated by PolymorphismBoxing, e.g. `PolymorphismBoxing.TBoxedInt.name` (after casting/type annotating)

Reply:
Thanks that was it

Reply:
Puh, thanks!